test/usage.js

```diff
@@ -2345,7 +2345,27 @@ describe('usage tests', () => {
       ])
     })
 
-    it('should call the correct console.log method', () => {
+    it('should print the help to stdout when no arguments were specified', () => {
```

Comment:
Isn't it `stderr` rather than `stdout` by default?

And is the wording `stdout`/`stderr` pertinent when we use `console.log`/`.error` which may have been redefined by the calling code?

Reply:
Fixed to "should print the help using console.error".